internal/bminventory/inventory.go

```diff
@@ -2443,6 +2450,13 @@ func (b *bareMetalInventory) v2UpdateClusterInternal(ctx context.Context, params
 		return nil, common.NewApiError(http.StatusInternalServerError, err)
 	}
 
+	err = b.updateClusterNetworkVMUsage(cluster, usages)
+	if err != nil {
+		return nil, err
```

Comment:
why do you abort the update here? For all other usages we do not abort the update process as it a nice-to-have feature

Reply:
Good point. You suggest adding a log and move on?

Reply:
Yes. But also add a test in the subsystem just to see that your logic works and does not break